stable/fluent-bit/README.md

```diff
@@ -17,7 +17,7 @@ To install the chart with the release name `my-release`:
 $ helm install --name my-release stable/fluent-bit
 ```
 
-When installing this chart on [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/), it's required to specify that so the DaemonSet will be able to mount the log files properly, make sure to append the _--set on\_minikube=true_ option at the end of the _helm_ command, e.g:
+When installing this chart on [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) or [MicroK8s](https://microk8s.io/), it's required to specify that so the DaemonSet will be able to mount the log files properly, make sure to append the _--set on\_minikube=true_ or _--set on\_microk8s=true_ option at the end of the _helm_ command, e.g:
```

Comment:
I kind of created this slippery slope, but am a bit uncomfortable with it too. adding a distro specific option for every distro that comes around could get ugly quickly... On the other hand, microk8s and minikube are kind of special? How many more do we foresee?

Reply:
See my comment above - a solution could also be to just update the documentation about how to mount the extra volumes.

Reply:
I guess I'm ok with adding specific, kick the tires style distro's like minikube/microk8s. as the goal of them is to make it super easy to introduce new users to k8s. So making it super simple to onboard them into fluent-bit's chart makes sense too. Having a config for every distro, not just the super quick/simple onboarding ones seems like too much work. This may come back to haunt us later.... :)